packages/react-sdk/docusaurus/docs/React/03-ui-components/03-video-theme.mdx

```diff
@@ -44,3 +44,87 @@ const App = () => {
   );
 };
 ```
+
+## Customization
+
+### Theme variables
+
+You can customize the **colors**, **border-radius settings**, and **icons** used by the built-in UI components.
+
+Customizing can be done using **CSS variables** defined in our [variables file](https://github.com/GetStream/stream-video-js/blob/main/packages/styling/src/_global-theme-variables.scss).
+
+```scss
+import '@stream-io/video-react-sdk/dist/css/style.css';
+
+// Use your own font
+html {
+  font-family: sans-serif;
+}
+
+.str-video {
+  --str-video__primary-color: #6002ee;
+  --str-video__secondary-color: #90ee02;
+  --str-video__text-color1: #282b2b;
+  --str-video__border-radius-circle: 15px;
+  --str-video__icon--download: url('base64 encoded SVG')
+}
+```
+
+### Light and dark themes
```

Comment:
There are some issues in light mode: https://www.notion.so/stream-wiki/8e8d9743b99f4379b24bd2415e5bfdbd?v=714cda30bb2d433fa7151a33556811c5&p=51a2060477b94d2ebc239aae236fa05a&pm=c - I don't think we will have the capacity to fix these before the beta release